assets/templates/partials/banners/beta.tmpl

```diff
@@ -1,3 +1,21 @@
+{{ if eq .Type "search" }}
+<div class="ons-phase-banner">
+  <div class="ons-container">
+    <div class="ons-grid ons-grid--flex ons-grid--gutterless ons-grid--vertical-center ons-grid--no-wrap">
+      <div class="ons-grid__col ons-col-auto ons-u-flex-no-grow ons-u-flex-no-shrink">
+        <h3 class="ons-phase-banner__badge">{{ localise "Beta" .Language 1 }}</h3>
+      </div>
+        <span class="ons-u-fs-s ons-u-m-no">{{ localise "BetaSearchFeedback" .Language 1 | safeHTML }}</span>
```

Comment:
Might make more sense to have your conditional `{{ if eq .Type "search" }}` here. There's not any difference between the containers to justify breaking them out as much as you have